kinto/core/resource/model.py

```diff
@@ -110,6 +111,12 @@ def delete_records(self, filters=None, parent_id=None):
             comparison (see `kinto.core.utils.COMPARISON`). All filters
             are combined using *AND*.
         :type filters: list of :class:`kinto.core.storage.Filter`
+        :param sorting: XXX
```

Comment:
:param filters: Optionally filter the records by their attribute.
             Each filter in this list is a tuple of a field, a value and a
             comparison (see `kinto.core.utils.COMPARISON`). All filters
             are combined using *AND*.
         :type filters: list of :class:`kinto.core.storage.Filter`
 
         :param sorting: Optionnally sort the records by attribute.
             Each sort instruction in this list refers to a field and a
             direction (negative means descending). All sort instructions are
             cumulative.
         :type sorting: list of :class:`kinto.core.storage.Sort`
 
         :param pagination_rules: Optionnally paginate the list of records.
             This list of rules aims to reduce the set of records to the current
             page. A rule is a list of filters (see `filters` parameter),
             and all rules are combined using *OR*.
         :type pagination_rules: list of list of
             :class:`kinto.core.storage.Filter`
 
         :param int limit: Optionnally limit the number of records to be
            retrieved.